CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 1.5.2 (2024-09-16)
+
+- Removed go mod replacements to allow `crd2pulumi` to be installed with `go install` command.
```

Comment:
Probably not worth mentioning since the replace was only temporary. I'd be surprised if anyone noticed or cared.